sphinx_immaterial/apidoc/format_signatures.py

```diff
@@ -275,9 +641,29 @@ def merge_info(
 ClangFormatStyle = Union[str, Dict[str, Any]]
 
 
+@pydantic.dataclasses.dataclass
+class BlackFormatStyle:
+    line_length: Optional[int] = None
+    """Line length limit.
+
+    Defaults to the value of :objconf:`wrap_signatures_column_limit`.
+    """
+
+    string_normalization: bool = True
+    """Indicates whether to normalize quotes."""
```

Comment:
```suggestion
    """Indicates whether to normalize quotes. This corresponds to the inverse of Black
    CLI's `--skip-string-normalization <https://black.readthedocs.io/en/stable/
    usage_and_configuration/the_basics.html#s-skip-string-normalization>`_ option."""
```

Unfortunately, they use a `<h3>` element in black docs, so we can't use intersphinx + `:std:option:'skip-string-normalization'`.